source/php/Submission.php

```diff
@@ -169,7 +170,7 @@ public function submit()
         }
         // Send auto reply
         if (get_field('autoreply', $_POST['modularity-form-id'])) {
-            $this->autoreply($fromEmail, $submission, $fromEmail);
+            $this->autoreply($fromEmail, $submission, $fromHelsingborg);
```

Comment:
Do not use keyword "Helsingborg". Out products should be separated from the keyword helsingborg.